docs/spec/staking/definitions and examples.md

```diff
@@ -108,7 +108,41 @@ worth more as share-to-atom-ex-rate is now worth 50/45 Atoms per share. Therefor
 
 `delegatorCoins = 10 (delegator shares) * 1 (delegator-share-to-global-share-ex-rate) * 50/45 (share-to-atom-ex-rate) = 100/9 Atoms`  
    
-
+### Inflation provisions
+
+Validator provisions are minted on an hourly basis (the first block of a new
+hour).  The annual target of between 7% and 20%. The long-term target ratio of
+bonded tokens to unbonded tokens is 67%.  
+    
+The target annual inflation rate is recalculated for each previsions cycle. The
+inflation is also subject to a rate change (positive of negative) depending or
+the distance from the desired ratio (67%). The maximum rate change possible is
+defined to be 13% per year, however the annual inflation is capped as between
+7% and 20%.
```

Comment:
Maybe also note that because its 13% per year, it would take an entire year for the inflation to move from its minimum threshold to it's maximum (that is how we came up with the 13% rate)